src/rpc/util.cpp

```diff
@@ -13,6 +13,12 @@
 
 #include <tuple>
 
+/**
+ * String used to describe UNIX epoch time in documentation, factored out to a
+ * constant for consistency.
+ */
+const std::string UNIX_EPOCH_TIME = "UNIX epoch time";
```

Comment:
It is kinda common knowledge, but wouldn't something like "UNIX epoch time (seconds since Jan 1 1970 GMT)" still be better?

Reply:
See http://www.erisian.com.au/bitcoin-core-dev/log-2019-11-26.html#l-368 to 373.

Reply:
the motivation here was: it is not within the domain of our help to explain what is meant here
various programming languages provide APIs that use epoch times, that's why it's used here, most developers will be better off treating it as an opaque value instead of trying to roll their own time handling
(and if  someone really want to know *what it is*, that's great, but it's easy enough to find information on Wikipedia and such)

Reply:
Yes. I probably should have mentioned that I agree with this motivation which is why the PR is proposed this way... hmmm it may be good to explain the rationale (and to use `UNIX_EPOCH_TIME` when referring to Unix Time in RPC help) in the "RPC interface guidelines" section of doc/developer-notes.md. Will propose.

Reply:
extending scope is easy, decreasing scope is always an uphill battle :smile:

Reply:
attempted an explanation at the bottom of developer-notes.md